packages/perseus/src/widgets/sorter/sorter-validator.test.ts

```diff
@@ -0,0 +1,41 @@
+import sorterValidator from "./sorter-validator";
+
+import type {Rubric, UserInput} from "./sorter.types";
+
+describe("SorterValidator", () => {
+    it("gives points when the answer is correct", () => {
```

Comment:
Could we define these tests rather as the description of what makes the answer correct or incorrect? Maybe something like: 

```js
   it("is correct when the user input values are in the same order as the rubric defines"...
```

When I read test names, I think if I'd have some idea of what broke when it fails. Just stating "is correct" can be said for all widget validators and doesn't point to what the test is actually validating.

Reply:
Updated the test names. Thank you! :)